packages/esm-patient-list-management-app/src/config-schema.ts

```diff
@@ -14,7 +14,7 @@ export const configSchema = {
   patientListsToShow: {
     _type: Type.Number,
     _description: 'The UUID of the `My List` cohort type',
-    _default: '10',
+    _default: 10,
```

Comment:
Don't know why this value was originally provided as a string.

Reply:
The description here is also weird? Neither 10 nor '10' is a UUID.

Reply:
Copypasta strikes again